workbench-core/infrastructure/src/workbenchCognito.ts

```diff
@@ -59,7 +59,6 @@ const userPoolClientDefaults: UserPoolClientOptions = {
   },
   authFlows: {
     userSrp: true,
-    userPassword: true,
```

Comment:
Have you verified this change doesn't affect SWB?

Reply:
I've recreated fresh environment for SWB and integration tests are passing. Are there any additional checks that I should execute?